pay-api/src/pay_api/services/statement_settings.py

```diff
@@ -197,3 +200,25 @@ def _get_end_of(frequency: StatementFrequency):
         if frequency == StatementFrequency.MONTHLY.value:
             end_date = get_first_and_last_dates_of_month(today.month, today.year)[1]
         return end_date
+
+    @classmethod
+    def find_accounts_settings_by_frequency(cls,
+                                            valid_date: datetime,
+                                            frequency: StatementFrequency,
+                                            from_date=None,
```

Comment:
Will fix this next PR, probably should indicate it's `statement_from` and `statement_to`